qt/python/mantidqt/plotting/test/test_functions.py

```diff
@@ -326,6 +326,17 @@ def _compare_errorbar_labels_and_title(self):
             # Compare title
             self.assertEqual(ax.get_title(), err_ax.get_title())
 
+    def test_colorbar_limits_not_default_values_on_surface_plot_with_monitor(self):
+        ws = CreateSampleWorkspace(NumMonitors=1)
+        fig = plt.figure()
+        plot_surface([ws], fig=fig)
+        ax = fig.get_axes()
+        cmin, cmax = ax[0].collections[0].get_clim()
+
+        # the colorbar limits default to +-0.1 when it can't find max and min of array
```

Comment:
More of a nit, we should know the min / max of `CreateSampleWorkspace` can we check for that instead as if the colorbars starts returning `NAN` this test would still pass?

Reply:
This is what we tried first but it was a real pain because the colorbar limits are calculated funny. If the max value in the data was `0.005`, the upper limit was `0.0049...`. It's also different between OS's, which is why the `assertAlmostEqual` failed on jenkins [for this commit](https://github.com/mantidproject/mantid/pull/29119/commits/9537ddb6273f1926beb9c0593211c30c14b2c9d3).

Reply:
Is this because MPL produces different results on each platform then?

Reply:
Yeah I think so, in some cases the max value in the data was (slightly) more than the colorbar limit

Reply:
Okay lets leave this for the moment, until our MPL major versions sync back up since it's still doing a valid test at the moment